name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/Direkt1822BankPDFExtractor.java

```diff
@@ -144,6 +144,11 @@ private void addDividendeTransaction()
                 .section("date")
                 .match("^Den Betrag buchen wir mit Wertstellung (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) .*$")
                 .assign((t, v) -> t.setDateTime(asDate(v.get("date"))))
+                
+                // Ex-Tag 29.11.2016
+                .section("exDate").optional()
+                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}).*")
```

Comment:
Is `.*` really needed?

Reply:
it's needed as we have 3 different types of Ex-Date-Entries. 

I added the second section including the note and edited the commentaries:
`// Ex-Tag 14.12.2017 Herkunftsland Irland
                // Ex-Tag 10.03.2021 fonds) 0,042000000 EUR
                .section("exDate").optional()
                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}).*")
                .assign((t,v) -> t.setExDate(asDate(v.get("exDate")).toLocalDate()))
                
                // Ex-Tag 09.02.2021 Art der Dividende Quartalsdividende
                .section("exDate", "note").optional()
                .match("^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4}) Art der Dividende (?<note>.*)$")
                .assign((t,v) -> {
                    t.setExDate(asDate(v.get("exDate")).toLocalDate());
                    t.setNote(v.get("note").trim());
                })`